### PR TITLE
fix shadow in BussinesCard and itemList and added new Headers

### PR DESCRIPTION
**Pull request by LiorBraginsky**
simulator screen shot - iphone 6 - 2018-06-21 at 17 29 52
simulator screen shot - iphone 6 - 2018-06-21 at 17 29 50
simulator screen shot - iphone 6 - 2018-06-21 at 17 29 33
simulator screen shot - iphone 6 - 2018-06-21 at 17 29 30
simulator screen shot - iphone 6 - 2018-06-21 at 17 24 16

### CODE REVIEW

File: src/screens/Services/index.js [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
-		headerStyle: {
-			borderBottomWidth: 0,
-			backgroundColor: '#eff2f9'
-		}
+        headerTitle: <Image
+            style={{
+                flex: 1,
+                marginLeft: Platform.OS === 'ios' ? 0 : 50,
+                width: Platform.OS === 'ios' ? 80 : 60,
+                height: Platform.OS === 'ios' ? 35 : 35,
```
Comment (po4erk) [inline src/screens/Services/index.js]: why here Platform.OS if value: 35 for all platform?

File: src/screens/Profile/index.js [hunk 2]
```diff
@@ -8,24 +8,21 @@ import {
...
-			name="cart"
-			style={{
-				marginRight: 20
-			}}
-		/>,
+        headerTitle: <Image
+            style={{
+                flex: 1,
+                width: Platform.OS === 'ios' ? 80 : 60,
+                height: Platform.OS === 'ios' ? 35 : 35,
```
Comment (po4erk) [inline src/screens/Profile/index.js]: 35 for all

File: src/screens/OrderHistory/index.js [hunk 3]
```diff
@@ -18,25 +18,23 @@ import {
...
-      onPress={() => alert('It will navigate you to the order!')}
-      name="cart"
-      style={{
-        marginRight: 20
-      }}
+        style={{
+            flex: 1,
+            marginLeft: Platform.OS === 'ios' ? 0 : 50,
+            width: Platform.OS === 'ios' ? 80 : 60,
+            height: Platform.OS === 'ios' ? 35 : 35,
```
Comment (po4erk) [inline src/screens/OrderHistory/index.js]: 35 for all

File: src/components/UI/HeaderLogo.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Platform, Image } from 'react-native';
+
+export const HeaderLogo =
+  <Image
+    style={{
+      flex: 1,
+      paddingBottom: Platform.OS === 'ios' ? 5 : 0,
+      width: Platform.OS === 'ios' ? 80 : 60,
+      height: Platform.OS === 'ios' ? 35 : 35,
```
Comment (po4erk) [inline src/components/UI/HeaderLogo.js]: 35 for all

**Comment by LiorBraginsky** [comment]
fix height added

File: src/screens/BusinessList/index.js [hunk 5]
```diff
@@ -6,6 +6,7 @@ import { TextTitle, TextSubtitle } from '/components/Typography/Text'
 import ButtonBlock from '/components/UI/ButtonBlock'
 import BusinessCard from '/components/UI/BusinessCard'
 import styles from './styles'
+import {HeaderLogo} from '../../components/UI/HeaderLogo';
```
Comment (olbesp) [inline src/screens/BusinessList/index.js]: Use absolute paths everywhere.

File: src/screens/BusinessAddItem/ItemsList.js [hunk 6]
```diff
@@ -16,16 +16,28 @@ const styles = StyleSheet.create({
...
     flexWrap: 'wrap',
+
   },
   item: {
     height: 180,
-    width: '45%',
-    margin: '2.5%',
+    width: '100%',
+  },
+  shadowBlock:{
```
Comment (olbesp) [inline src/screens/BusinessAddItem/ItemsList.js]: We don't need shadows on this screen
asdasd

**Pull request merged**